Resources/Prototypes/DeltaV/Entities/Objects/Devices/pda.yml

```
@@ -40,6 +40,123 @@
     scanningEndSound:
       path: "/Audio/Items/Medical/healthscanner.ogg"
 
+- type: entity
+  parent: BasePDA
+  id: CJPDA
+  name: chief justice PDA
+  description: Whosoever bears this PDA is the law.
+  components:
+  - type: Sprite
+    sprite: DeltaV/Objects/Devices/pda.rsi
+    layers:
+    - map: [ "enum.PdaVisualLayers.Base" ]
+    - state: "light_overlay"
+      map: [ "enum.PdaVisualLayers.Flashlight" ]
+      shader: "unshaded"
+      visible: false
+    - state: "id_overlay"
+      map: [ "enum.PdaVisualLayers.IdLight" ]
+      shader: "unshaded"
+      visible: false
+  - type: Pda
+    id: CJIDCard
```

Doesn't follow naming convention; + who is CJ?
```suggestion
    id: ChiefJusticeIDCard
```